src/test_l1_tvSettings.c

```diff
@@ -390,21 +379,26 @@ void test_l1_tvSettings_negative_RegisterVideoFormatChangeCB (void)
 	result = TvInit();
 	UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
 
-	/* Step 03: Calling tvsettings RegisterVideoFormatChangeCB and expecting the API to return success*/
+	 /* Step 03: Calling tvsettings RegisterVideoFormatChangeCB and expecting the API to return success*/
+	result = RegisterVideoFormatChangeCB(NULL);
+	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_INVALID_PARAM);
+
+	/* Step 04: Calling tvsettings RegisterVideoFormatChangeCB and expecting the API to return success*/
 	callbackData.cb = NULL;
 	result = RegisterVideoFormatChangeCB(&callbackData);
-    UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_INVALID_PARAM);
+	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_INVALID_PARAM);
 
-	/* Step 04: Calling tvsettings RegisterVideoFormatChangeCB and expecting the API to return success*/
+	/* Step 05: Calling tvsettings RegisterVideoFormatChangeCB and expecting the API to return success*/
 	callbackData.userdata = NULL;
+	callbackData.cb = tvVideoFormatChangeHandler;
 	result = RegisterVideoFormatChangeCB(&callbackData);
 	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_INVALID_PARAM);
 
-	/* Step 05: Calling tvsettings initialization and expecting the API to return success */
+	/* Step 06: Calling tvsettings initialization and expecting the API to return success */
 	result = TvTerm();
 	UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
 
-	/* Step 06: Calling tvsettings RegisterVideoFormatChangeCB and expecting the API to return success*/
+	/* Step 07: Calling tvsettings RegisterVideoFormatChangeCB and expecting the API to return success*/
 	callbackData.cb = tvVideoFormatChangeHandler;
```

Comment:
how about userData here? It should be set to valid value.

Reply:
should be valid data in non negative cases.

Reply:
Done